### DISCUSSION TITLE
Archetypes cloud AWS & server - Equivalent needed

### DISCUSSION THREAD

**Discussion by damienfernandes** [Boavizta/boaviztapi#293] [Q&A]
Hello,

I would like to know data (and add to referential if you want) some new AWS instance type not already referenced:

- [ ] db.t4g.medium _> I see for other db.XX.yy we use server platform_aws_XX => ie platform_aws_t4g ?_
- [ ] mq.m5.large _> equivalent to m5.large ? Can I use server platform_aws_m5-b ?_
- [ ] kafka.m5.large _> equivalent to m5.large  ? Can I use server platform_aws_m5-b ? _
- [ ] kafka.t3.small _> equivalent to t3.small  ? Can I use server platform_aws_t3 ?_
- [ ] cache.t4g.small _> equivalent to t4g.small ? Can I use server platform_aws_t4g ?_
- [ ] r6g.large.search _> equivalent to r6g.large.elasticsearch ? Can I use server r6g.metal ?_

- [ ] Aurora PostgreSQL serverless : db.serverless _> no idea about equivalent_
- [ ] M10 _> no idea about equivalent_ 

Feel free to discuss about each archetypes separtely.
Thanks in advance <3

Referencial links:
- Link to existing AWS archetypes cloud : [boaviztapi/data/archetypes/cloud/aws.csv](https://github.com/Boavizta/boaviztapi/blob/main/boaviztapi/data/archetypes/cloud/aws.csv)
- Link to existing archetypes server : [boaviztapi/data/archetypes/server.csv](https://github.com/Boavizta/boaviztapi/blob/main/boaviztapi/data/archetypes/server.csv)

FYI : @demeringo

**Comment by demeringo** [comment]
Most of the instances listed above look from their name like specialized instances (I _imagine_ that they are standard instances but come preconfigured with software for specific tasks).

For example AWS seems to provide "brokers" instances for MQ: https://docs.aws.amazon.com/amazon-mq/latest/developer-guide/broker.html. 
>A broker is a message broker environment running on Amazon MQ. It is the basic building block of Amazon MQ. The combined description of the broker instance class (m5, t3) and size (large, micro) is a broker instance type (for example, mq.m5.large).
According to the doc, I believe that it is realistic to approximate them by their standard instance type.

But we should verify this assertion for other types of specialized instances... and more importantly document the principle if we add the data to Boavizta API.
